notebooks/recommender-system.ipynb

```diff
@@ -0,0 +1,843 @@
+{
```

Comment:
<p>Do you need floats here? <code>True</code>  and <code>False</code>  are 1 and 0 integer objects</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cooperrc/uconn_numpy_tutorials/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>